operations/validate.go

```diff
@@ -35,13 +34,17 @@ func Validate() cli.Command {
 		}, cli.StringFlag{
 			Name:  joinFlagNames(projectFlagName, "p"),
 			Usage: "specify project identifier in order to run validation requiring project settings",
+		}, cli.BoolFlag{
```

Comment:
One thing we can do to make users more aware of this pending upgrade is to include some text output when running `validate` saying that you'll need to ensure your validation still passes with the upgraded flag.

Reply:
I hesitate to do this because a bunch of users script using `evergreen validate` and may break if they use `--quiet` and expect no output. My expectation is that if they don't know about the YAML upgrade already, they're unlikely to happenstance use `evergreen validate` manually (aside from in a script, where they won't check the output).